mrkm00 - sp2

michalmrkus opened the pull request:
https://eso.vse.cz/~mrkm00/sp2/

michalmrkus committed:
mrkm00 - sp2

michalmrkus committed:
version 2.0

Final version without style

michalmrkus committed:
version 2.1

Basic style added

nvbach91 commented on www/mrkm00/sp2/assets/js/main.js:
```diff
@@ -0,0 +1,143 @@
+    </div>
+    `);
+
+    $(document).ready(function () {
```
document.ready tady?

nvbach91 commented on www/mrkm00/sp2/assets/js/main.js:
```diff
@@ -0,0 +1,143 @@
+
+    $(document).ready(function () {
+        if (localStorage.getItem("myFavorites") !== null) {
+            teamsInStorage = [];
```
const?

nvbach91 commented on www/mrkm00/sp2/assets/js/main.js:
```diff
@@ -0,0 +1,143 @@
+                const newElement = addFavoriteTeam(chosenData);
+                listFavorites.append(newElement);
+        */
+
```
do localStorage je lepsi ukladat jako objekt protoze budete ty tymy take mazat, jako klice jednotlivych tymu pouzijte jejich `idTeam`, potom se da jednoduse napsat `delete teamsInStorage[idTeam]` a pri cteni pouzijete `Object.keys()`

nvbach91 commented on www/mrkm00/sp2/assets/js/main.js:
```diff
@@ -0,0 +1,143 @@
+                listFavorites.append(newElement);
+        */
+
+        favorite.push(chosenData);
```
musite tady jeste zkontrolovat jestli ten tym uz je v seznamu, protoze nema smysl ho pridavat dvakrat

michalmrkus committed:
version 2.2

Zamezení duplikátů v seznamu oblíbených týmů + doopraveny připomínky ohledně document(ready) a chybějící const.
